cvat/apps/dataset_manager/formats/transformations.py

```diff
@@ -32,3 +35,16 @@ def transform_item(self, item):
                 z_order=ann.z_order))
 
         return item.wrap(annotations=annotations)
+
+class EllipsesToMasks(ItemTransform):
+    @staticmethod
+    def convert_ellipse(ellipse, img_h, img_w):
+        cx, cy, rightX, topY = ellipse.points
+        rx, ry = rightX - cx, cy - topY
+        shape = (cx - rx, cy - ry, cx + rx, cy + ry)
+        mask = Image.new('1', (img_w, img_h))
+        drawer = ImageDraw.Draw(mask)
+        drawer.ellipse(shape, fill = 1)
```

Comment:
I suspect, opencv may be a bit faster, did you try to check it?

Reply:
I'll investigate this point

Reply:
Yes, OpenCV about 2.5 times faster, I updated the code.